thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -241,17 +240,23 @@ class CIndexReadSlaveBase : public CSlaveActivity
             Owned<IDelayedFile> lfile = queryThor().queryFileCache().lookup(*this, logicalFilename, part);
             Owned<IKeyManager> klManager;
 
-            bool remoteKey = !localKey && !seekGEOffset && (!rfn.isLocal() || getOptBool("forceDafilesrv"));
-            if (remoteKey && getOptBool("remoteKeyFilteringEnabled"))
-                klManager.setown(createRemoteKeyManager(filePath.str(), fixedDiskRecordSize, lfile));
-            else
+            unsigned crc=0;
+            part.getCrc(crc);
+
+            if ((localKey && partDescs.ordinality()>1) || seekGEOffset) // for now at least, no remote key support if stepping or merging
```

Comment:
It is correct (as in intended), only want to use key merger if >1 part and local.